src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishANetCoreAppForTelemetry.cs

```diff
@@ -83,7 +85,7 @@ void It_collects_crossgen2_publishing_properties(string targetFramework)
                 .Contain(
                     "{\"EventName\":\"PublishProperties\",\"Properties\":{\"PublishReadyToRun\":\"True\",\"PublishTrimmed\":\"null\",\"PublishSingleFile\":\"null\"}")
                 .And.Contain(
-                    "{\"EventName\":\"ReadyToRun\",\"Properties\":{\"PublishReadyToRunUseCrossgen2\":\"True\",")
+                    "{\"EventName\":\"ReadyToRun\",\"Properties\":{\"PublishReadyToRunUseCrossgen2\":\"true\",")
```

Comment:
@LakshanF @vitek-karas @agocke Also see this capitalization change.

Reply:
@trylek as someone who actually knows things about R2R/CrossGen.

Reply:
The property is compared against in MSBuild scripts that use case-insensitive string comparison so casing of the <code>true</code> constant is unimportant and is purely a matter of visual coding style.